nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/CandidateApprovalTest.java

```diff
@@ -377,19 +383,42 @@ void shouldNotResetApprovalWhenOtherCreatorBecomesVerified() {
     assertThat(2, is(equalTo(updatedApprovals.size())));
   }
 
+  @Test
+  void shouldNotResetApprovalWhenOtherCreatorBecomesVerified2() {
```

Comment:
"2" implies parameterisation is called for?

Reply:
Actually a duplicate test case, can remove one of them...